e2e/tests/settings/smoke-test.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { test, expect, Page } from '@playwright/test';
+// eslint-disable-next-line import/extensions
+import { resetDatabaseAndImportDataFromPath } from '../../scripts/dts-import';
+import { login } from '../../utils/login';
+import { describeOnCondition } from '../../utils/shared';
+
+const edition = process.env.STRAPI_DISABLE_EE === 'true' ? 'CE' : 'EE';
+
+const headerVisible = async (page: Page, name: string) => {
+  await page.waitForSelector(`:is(h1, h2, h3, h4, h5, h6):has-text("${name}")`);
+  return page.getByRole('heading', { name }).isVisible();
+};
+
+test.describe('Settings', () => {
+  test.beforeEach(async ({ page }) => {
+    await resetDatabaseAndImportDataFromPath('./e2e/data/with-admin.tar');
+    await page.goto('/admin');
+    await login({ page });
+  });
+
+  test('every expected feature is displayed', async ({ page }) => {
+    await page.getByRole('link', { name: 'Settings' }).click();
+    expect(page.getByRole('heading', { name: 'Overview' })).toBeVisible();
```

Comment:
It looks like this is failing on github, we probably need the headerVisible fix here too